[2024-10-10T06:39:41.841+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-10T06:39:41.878+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: api_dag.is_api_active scheduled__2024-10-09T00:00:00+00:00 [queued]>
[2024-10-10T06:39:41.896+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: api_dag.is_api_active scheduled__2024-10-09T00:00:00+00:00 [queued]>
[2024-10-10T06:39:41.898+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-10T06:39:41.932+0000] {taskinstance.py:2330} INFO - Executing <Task(HttpSensor): is_api_active> on 2024-10-09 00:00:00+00:00
[2024-10-10T06:39:41.946+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=273) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-10T06:39:41.951+0000] {standard_task_runner.py:64} INFO - Started process 274 to run task
[2024-10-10T06:39:41.946+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'api_dag', 'is_api_active', 'scheduled__2024-10-09T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/api_dag.py', '--cfg-path', '/tmp/tmp5_86h_2h']
[2024-10-10T06:39:41.953+0000] {standard_task_runner.py:91} INFO - Job 4: Subtask is_api_active
[2024-10-10T06:39:41.991+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-10-10T06:39:42.074+0000] {task_command.py:426} INFO - Running <TaskInstance: api_dag.is_api_active scheduled__2024-10-09T00:00:00+00:00 [running]> on host ec5690b705ef
[2024-10-10T06:39:42.471+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='api_dag' AIRFLOW_CTX_TASK_ID='is_api_active' AIRFLOW_CTX_EXECUTION_DATE='2024-10-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-09T00:00:00+00:00'
[2024-10-10T06:39:42.474+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-10T06:39:42.600+0000] {baseoperator.py:400} WARNING - HttpSensor.execute cannot be called outside TaskInstance!
[2024-10-10T06:39:42.602+0000] {http.py:137} INFO - Poking: posts/
[2024-10-10T06:39:42.625+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-10T06:39:42.655+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/http/sensors/http.py", line 164, in execute
    super().execute(context=context)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/sensors/base.py", line 264, in execute
    raise e
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/sensors/base.py", line 246, in execute
    poke_return = self.poke(context)
                  ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/http/sensors/http.py", line 158, in poke
    raise exc
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/http/sensors/http.py", line 139, in poke
    response = hook.run(
               ^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/http/hooks/http.py", line 167, in run
    session = self.get_conn(headers)
              ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/http/hooks/http.py", line 108, in get_conn
    conn = self.get_connection(self.http_conn_id)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/hooks/base.py", line 83, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/connection.py", line 519, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `http_default` isn't defined
[2024-10-10T06:39:42.673+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=api_dag, task_id=is_api_active, run_id=scheduled__2024-10-09T00:00:00+00:00, execution_date=20241009T000000, start_date=20241010T063941, end_date=20241010T063942
[2024-10-10T06:39:42.741+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 4 for task is_api_active (The conn_id `http_default` isn't defined; 274)
[2024-10-10T06:39:42.778+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 1
[2024-10-10T06:39:42.809+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/models/baseoperator.py:1297 AirflowProviderDeprecationWarning: Call to deprecated class SimpleHttpOperator. (Class `SimpleHttpOperator` is deprecated and will be removed in a future release. Please use `HttpOperator` instead.)
[2024-10-10T06:39:42.874+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-10T06:39:43.020+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
